MySQL/legos/mysql_kill_long_query/mysql_read_query.py

```diff
@@ -0,0 +1,51 @@
+##
+##  Copyright (c) 2021 unSkript, Inc
+##  All rights reserved.
+##
+import pprint
+from typing import List,Any
+from pydantic import BaseModel, Field
+from tabulate import tabulate
+
+
+class InputSchema(BaseModel):
+    query: str = Field(
+        title='Read Query',
+        description='MySQL get query.')
+    params: List = Field(
+        None,
+        title='Parameters',
+        description='Parameters to the query in list format. For eg: [1, 2, "abc"]')
+

```

Comment:
I don't think this is the correct lego. Perhaps you have copied the wrong files. Please relook thru this diff.